### DISCUSSION TITLE
Night vision by the book

### DISCUSSION THREAD

**Discussion by Yuffyyy** [Jagusti/fvtt-wfrp4e-gmtoolkit#187] [Ideas]
Hi there !
I love your module and was wondering if it could integrate the rule form night vision that is : expand vision provided by all lights sources by 20yds per level of night vision ? (sounds really complicated to create as a feature but Dev is like witchcraft for me so I cannot really fathom what can be done and what cannot ^^")

**Comment by Jagusti** [comment]
Thanks @Yuffyyy! 

So, the version of Night Vision that is baked into GM Toolkit is to extend the illumination distance of light sources that the character is set to use by 20yd/level.  

> Example: Use the Set Token Vision & Light macro on a character with 1 level of Night Vision talent to set their Sight to Night Vision and Light to Candle (5yds Bright, 10yds dim). They will then 30 yds visibility, of which the first 5yds will be brightened by the candle.

In theory, the illumination could be extended for any (scene-based) light source that the token representing the character has line of sight for. However, that would then change the effective illumination for all characters, even if they didn't have the Night Vision talent or trait. AFAIK there isn't a way to adjust illumination that is not on a character just for that perceiving character.

**Reply by Yuffyyy** [reply]
I think I do understand ! Thank you for the reply !

**Reply by Jagusti** [reply]
No worries. Thanks for the suggestion. I'll keep it in mind as I'm working on lighting chairs. I think it would be great to do if possible. There have been a lot of updates to the lighting and vision engine that I need to catch up on.

**Reply by Yuffyyy** [reply]
It would be so coooooool ! Fingers crossed ♥
